<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>554</x>
      <y>391</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="Buttons">
     <property name="spacing">
      <number>15</number>
     </property>
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QPushButton" name="SaveButton">
       <property name="text">
        <string>Зберегти</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ExitButton">
       <property name="text">
        <string>Вийти</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="AccountList">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>60</y>
      <width>231</width>
      <height>301</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>160</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="lineName">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <italic>true</italic>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineSurn">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <italic>true</italic>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="linePasp">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <italic>true</italic>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineBank">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <italic>true</italic>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="lineDeposit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>280</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineWithdraw">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>320</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineNumCard">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>360</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonCreateAcc">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>0</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Створити 
новий акаунт</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>531</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonDeposit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Депозит</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonWithdraw">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Вивести</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonTransfer">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Перевести на</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonAddCard">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>220</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Додати картку</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineSum">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>390</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonCardInfo">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>390</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Інформація про картку</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonDeleteCard">
    <property name="geometry">
     <rect>
      <x>402</x>
      <y>220</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Видалити картку</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ім'я</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSurn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Прізвище</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPasp">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Номер пасп.</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBank">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Банк</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listCard">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>260</y>
      <width>231</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonDeleteAcc">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>0</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Видалити
 акаунт</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Операції над картками</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>505</x>
      <y>0</y>
      <width>51</width>
      <height>451</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>lineDeposit</zorder>
   <zorder>lineWithdraw</zorder>
   <zorder>lineNumCard</zorder>
   <zorder>ButtonCreateAcc</zorder>
   <zorder>line</zorder>
   <zorder>AccountList</zorder>
   <zorder>ButtonDeposit</zorder>
   <zorder>ButtonWithdraw</zorder>
   <zorder>ButtonTransfer</zorder>
   <zorder>ButtonAddCard</zorder>
   <zorder>lineSum</zorder>
   <zorder>ButtonCardInfo</zorder>
   <zorder>ButtonDeleteCard</zorder>
   <zorder>labelName</zorder>
   <zorder>labelSurn</zorder>
   <zorder>labelPasp</zorder>
   <zorder>labelBank</zorder>
   <zorder>listCard</zorder>
   <zorder>ButtonDeleteAcc</zorder>
   <zorder>label</zorder>
   <zorder>line_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>823</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
